src/test/regress/citus_tests/run_test.py

```diff
@@ -199,6 +199,7 @@ def extra_tests(self):
         repeatable=False,
     ),
     "multi_prepare_plsql": TestDeps("base_schedule"),
+    "pg15": TestDeps("base_schedule"),
```

Comment:
why do we need this item?

Reply:
This is because this PR needs to update pg15.sql but the flaky test detection script didn't know how to "test" pg15.sql against test-flakyness properly.